sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -1,103 +1,151 @@
 <template>
-  <modal-dialog-base
-    title="Application already in progress"
-    dialogType="warning"
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-  >
-    <template v-slot:content>
-      <v-container>
-        <form>
-          <p>There is already a draft of an application in progress.</p>
-          <p>Please continue your draft application or cancel it.</p>
-        </form>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <footer-buttons
-        secondaryLabel="Close"
-        :showPrimaryButton="false"
-        @secondaryClick="dialogClosed"
-      />
-    </template>
-  </modal-dialog-base>
-  <student-page-container>
-    <v-sheet elevation="1" class="mx-auto">
-      <v-container>
-        <formio
-          formName="studentapplication"
-          :data="initialData"
-          @loaded="formLoaded"
-          @customEvent="customEventCallback"
-        ></formio>
-      </v-container>
-    </v-sheet>
-  </student-page-container>
+  <v-form ref="startApplicationForm">
+    <modal-dialog-base
+      title="Application already in progress"
+      dialogType="warning"
+      :showDialog="showDialog"
+      @dialogClosed="closeDialog"
+    >
+      <template v-slot:content>
```

Comment:
There was a decision to try to use the shortcuts as much as possible.
That is why for events we use the `@`, for properties we use `:`, and for template name we use `#`, like `<template #content>`.

Reply:
ok. Thanks!